src/components/HelloWorld.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <div id="container">
+      <h1>Menu of Salads</h1>
+      <ul>
+        <li v-for="(item, index, price) in myJson" :key="index" :id="index">
+          {{ index + 1 }}. {{ item }}
+          <div class="lisupport">
+            <p v-if="index % 3 == 0">{{ (price = +15) }}</p>
+            <p v-else-if="item.startsWith('E')">{{ (price = +10) }}</p>
+            <p v-else-if="item.startsWith('R')">{{ (price = +10) }}</p>
+            <p v-else-if="item.startsWith('G')">{{ (price = +10) }}</p>
```

Comment:
można by to ująć w jednej linijce. Najlepiej ubrać w jakąś metodę :). Uniknąłbyś tej duplikacji kodu.